### ISSUE TITLE
[bug | 0.5.4] Recipes require valid power / time 

### ISSUE THREAD

**Issue by aria1th** [TED-inc/FabricQuarry#34]
Suggested : 
```
{
  "type": "techreborn:rolling_machine",
  "shaped": {
    "pattern": [
      " S ",
      " A ",
      " S "
    ],
    "key": {
      "A": {
        "item": "techreborn:advanced_alloy_plate"
      },
      "S": {
        "tag": "c:steel_plates"
      }
    },
    "result": {
      "item": "quarrymod:drill_tube",
      "count": 6
    }
  },
  "power": 1,
  "time": 1
}
```
**Mods versions:**
 - Quarry Reborn: 0.5.4
 - Tech Reborn: 5.1.0 Beta-7+
 - Fabric API: 0.46.2

**Comment by FirstMegaGame4** [comment]
You can create a pull request

**Comment by TED-inc** [comment]
Thanks for report! Fixed in 0.5.5
